feature(examples): add awardco connector (#266)
fivetran/fivetran_connector_sdk: README.md

- [apache_hive/using_pyhive](https://github.com/fivetran/fivetran_connector_sdk/tree/main/connectors/apache_hive/using_pyhive) - This example shows how you can sync data from Apache Hive by using Connector SDK and PyHive.
- [arango_db](https://github.com/fivetran/fivetran_connector_sdk/tree/main/connectors/arango_db) - This example demonstrates how to sync data from ArangoDB, a native multi-model database combining document, graph, and key-value capabilities. The connector syncs document collections (airports, points-of-interest) and edge collections (flights) with offset-based pagination and checkpointing. You need to provide your ArangoDB credentials for this example to work.
- [apache_hive/using_sqlalchemy](https://github.com/fivetran/fivetran_connector_sdk/tree/main/connectors/apache_hive/using_sqlalchemy) - This example shows how you can sync data from Apache Hive by using Connector SDK and SQLAlchemy with PyHive.

> - [awardco](https://github.com/fivetran/fivetran_connector_sdk/tree/main/connectors/awardco) - This example shows how you can sync data from Awardco by using Connector SDK. You will need to provide an API key for this example to work.

Reviewer: ```suggestion
- [awardco](https://github.com/fivetran/fivetran_connector_sdk/tree/main/connectors/awardco) - This example shows how to sync data from Awardco with the Connector SDK. You will need to provide an API key for this example to work.
```
